app/components/Exchange/MyOpenOrders.jsx

```diff
@@ -16,11 +17,11 @@ import {LimitOrder, CallOrder} from "common/MarketClasses";
 import {EquivalentValueComponent} from "../Utility/EquivalentValueComponent";
 import {MarketPrice} from "../Utility/MarketPrice";
 import FormattedPrice from "../Utility/FormattedPrice";
-const leftAlign = {textAlign: "left"};
+const leftAlign = {textAlign: "left !important"};
```

Comment:
CSS nightmare

Reply:
Yeah. I'm not sure if this should be required here. I think another !important from the CSS files was holding it back, which was another CSS nightmare.